superset/views/geocoding.py

```diff
@@ -305,13 +343,23 @@ def _add_lat_lon_columns(
         :param lon_column: The name of longitude column
         :raise SqlAddColumnException: When add column-SQL went wrong
         """
-        connection = db.engine.connect()
+        table = db.session.query(SqlaTable).filter_by(table_name=table_name).first()
+        database = (
```

Comment:
Does the table itself not contain a reference to the database?  Would make this call unnecessary, if not it's ok